doc/source/workflows/basics.rst

```diff
@@ -362,6 +370,8 @@ The following example shows how to implement the recursive ``factorial`` program
         if n == 1:
```

Comment:
Until line 361 above I was confused. I assumed "continuation" is a Python concept that deals with tail functions, but there doesn't seem to be such a concept in Python. It's only when I see line 375 below that I realize you are trying to set up context to show what a dynamic workflow looks like.

I wonder if you need to introduce the concept of "continuation" - instead just describe the semantics of the "continuation(...)" function and say where it can be used.